.github/workflows/conformance-aks.yaml

```diff
@@ -287,10 +288,20 @@ jobs:
         run: |
           mkdir -p cilium-junits
 
-      - name: Run connectivity test (${{ join(matrix.*, ', ') }})
+      - name: Run sequential connectivity test (${{ join(matrix.*, ', ') }})
```

Comment:
Are you going to remove this step?
Maybe I'm missing something but it looks like we are running both sequential and concurrent steps in the workflow.

Reply:
This is intentional. The first step (called sequential) runs all tests with the `seq-` prefix sequentially, the second step (called concurrent) runs all other tests (`!seq-`) concurrently. This exists because all tests with the `seq-` prefix are suspected to be flaky when running in concurrent mode. I took this from the e2e upgrade workflow. See this PR which introduced this pattern:

- https://github.com/cilium/cilium/pull/34806

Reply:
Oh, I missed that! Sorry and thank for the clarifications.